internal/provisioner/prometheus.go

```diff
@@ -0,0 +1,99 @@
+package provisioner
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mattermost/mattermost-cloud/internal/tools/aws"
+	"github.com/mattermost/mattermost-cloud/internal/tools/kops"
+	"github.com/mattermost/mattermost-cloud/model"
+	"github.com/pkg/errors"
+)
+
+type prometheus struct {
+	cluster     *model.Cluster
+	provisioner *KopsProvisioner
+	awsClient   aws.AWS
+	kops        *kops.Cmd
+}
+
+func newPrometheusHandle(cluster *model.Cluster, provisioner *KopsProvisioner, awsClient aws.AWS, kops *kops.Cmd) (*prometheus, error) {
+
```

Comment:
nit: extra newlines

Reply:
I don't know why this piece of the diff doesn't show as outdated but I have removed the extra newline